package 异常测试;

/**
 * @author chengzhen
 * @date 2020/5/14
 * @time 上午10:08
 */
public class ExceptionTest {
    public static void main(String[] args) {
    }

//    public static String TestException(){
//        throw new RuntimeException("测试后续返回是否有用");
//        return "";
//        //这里返回没用
//    }
//
//    public static String TestException2(){
//        return "";
//        throw new RuntimeException("测试后续返回是否有用");
//        //这里返回没用
//    }
}
